gramex/apps/admin2/index.html

```diff
@@ -187,12 +187,8 @@ <h4 class="alert-heading">User management is not enabled</h4>
       <div class="h1 text-danger">Work in progress</div>
 
     {% elif current_tab == 'nlg' %}
-      {% set action = handler.get_argument('action', '') %}
-      {% if action == 'edit' %}
-        {% include "../nlg/index.html" %}
-      {% else %}
-        {% include "../nlg/grmform.html" %}
-      {% end %}
+      {% set nlg_base = 'nlg/' %}
```

Comment:
Convention: use `nlg_base = 'nlg'` instead of `nlg/`. This allows the use of `{{ nlg_base }}/index.html` which is intuitive. A few other things become easier.